tests/foreman/sys/test_fam.py

```diff
@@ -235,11 +234,9 @@ def test_positive_run_modules_and_roles(module_target_sat, setup_fam, ansible_mo
     ]:
         pytest.skip(f"{ansible_module} module test lacks proper setup")
 
-    # Skip oVirt/RHV tests on IPv6 setups
-    if module_target_sat.network_type == NetworkType.IPV6 and ansible_module in [
-        'compute_profile_ovirt'
-    ]:
-        pytest.skip("oVirt/RHV is not properly set up in IPv6 environment")
+    # RHV/oVirt support removed in SAC for 6.18+
+    if ansible_module == "compute_profile_ovirt":
+        pytest.skip("oVirt/RHV support is removed for 6.18+")
```

Comment:
@lpramuk Instead of skipping it we could've removed `compute_profile_ovirt` from FAM_TEST_PLAYBOOKS, so it would not run for 6.18+

Reply:
No we couldn't. 
See #18756 for @rmynar pointing this out.